cmd/minikube/cmd/start.go

```diff
@@ -143,6 +144,16 @@ func runStart(cmd *cobra.Command, args []string) {
 		os.Setenv(k, v)
 	}
 
+	registrator, err := registration.DetectRegistrator(host.Driver)
+	if err != nil {
+		glog.Info("Distro doesn't support registration")
```

Comment:
Don't use glog at this level. If you want the user to see this, then use ftm.Println for now until we have a proper logging facade.